requirements.txt

```diff
@@ -1,5 +1,5 @@
-ansible==2.7.16
-jinja2==2.10.1
+ansible==2.8.10
```

Comment:
Centos8 requires Ansible 2.8?

I noticed that the `ansible_os_family` doesn't work in Ansible 2.7 for Ubuntu 20.04, so I guess a similar issue with Centos8?